docs/_docs/grpcapiconfiguration.md

```diff
@@ -61,7 +63,23 @@ The following is equivalent to the basic [usage example](usage.html) but without
     protoc -I. --grpc-gateway_out=logtostderr=true,paths=source_relative,grpc_api_configuration=path/to/your_service.yaml:./gen/go \
       your/service/v1/your_service.proto
     ```
-   
+
    This will generate a reverse proxy `gen/go/your/service/v1/your_service.pb.gw.go` that is identical to the one produced for the annotated proto.
 
+6. Generate the optional your_service.swagger.json
+
+    ```sh
+    protoc -I. --swagger_out=grpc_api_configuration=path/to/your_service.yaml:./gen/go \
+      your/service/v1/your_service.proto
+    ```
+
+    or using an OpenAPI configuration file
+
+    ```sh
+    protoc -I. --swagger_out=grpc_api_configuration=path/to/your_service.yaml,openapi_configuration=path/to/your_service_swagger.yaml:./gen/go \
+      your/service/v1/your_service.proto
+    ```
```

Comment:
```suggestion
    ```sh
    protoc -I . --swagger_out ./gen/go \
      --swagger_opt grpc_api_configuration=path/to/your_service.yaml \
      --swagger_opt openapi_configuration=path/to/your_service_swagger.yaml \
      your/service/v1/your_service.proto
    ```
```